assets/src/edit-story/constants.js

```diff
@@ -28,10 +28,13 @@ export const PAGE_NAV_PADDING = 60;
 export const PAGE_NAV_BUTTON_WIDTH = 40;
 export const PAGE_NAV_WIDTH = PAGE_NAV_PADDING + PAGE_NAV_BUTTON_WIDTH;
 
+export const PAGE_RATIO = 3 / 2;
```

Comment:
I hate to do it to you, but per some agreement we decided to always have ratio to mean `width / height`. Could you pls reverse?